.github/workflows/CI.yml

```diff
@@ -104,10 +107,11 @@ jobs:
 
       # Regression Testing - Test Execution
       - name: Regression Tests - Execution
+        timeout-minutes: 60
```

Comment:
What is the unit of this?

Reply:
`timeout-minutes: 60`
minutes

Reply:
Lol can't believe I didn't see that on the first glance, thanks!